packages/framework/client-logger/app-insights-logger/src/fluidAppInsightsLogger.ts

```diff
@@ -30,26 +30,93 @@ export interface FluidAppInsightsLoggerConfig {
 		 */
 		mode: "inclusive" | "exclusive";
 		/**
-		 * Controls the default filtering of log events by their category.
+		 * Controls the filtering of log events.
 		 * Leaving this undefined will be treated as an empty array.
+		 *
+		 * In order for the filters to be valid they must meet the following conditions:
+		 * 1. There must not be any two filters with the same `namespacePattern`.
```

Comment:
Remember that you'll need blank lines between entries here in order to have this treated as a list by TSDoc. You'll also need a blank line before `1.`, or the first list entry will appear on the same line as the preceding text.

Reply:
```suggestion
		 *
		 * 1. There must not be any two filters with the same `namespacePattern`.
```

Reply:
Updated.